plugins/scorecards/src/components/CheckResultRenderer.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CheckResult } from '@backstage/plugin-tech-insights-common';
+import React from 'react';
+import { BooleanCheck } from './BooleanCheck';
+
+export type CheckResultRenderer = {
+  type: string;
+  title: string;
+  description: string;
+  component: React.ReactElement;
+};
+
+export function defaultCheckResultRenderers(
+  value: CheckResult[],
+): CheckResultRenderer[] {
+  return [
+    {
+      type: 'json-rules-engine',
```

Comment:
I'm a little wary of making this one to one with the rules engine (though "type" could be changed easily enough) . In principle I would have thought that this could be used two different engines or similarly an engine might want two different types of renderer (though we're keeping things simple for now).

Essentially what do you think of `type: "boolean"` or even `type: "json-rules-engine:boolean"`

Reply:
I do think that is a fair point but wonder if we should be addressing multiple engines/different type of renderer in our initial PR for scorecards? I was under impression we are not doing multiple types of renderer per engine. Btw, I would be leaning towards type: "boolean" more though.

Reply:
yeah actually you can ignore this; This has to match data from the backend.

Reply:
the name is a tad bit awkward but it does imply that the result contains additional data (rules in this case) in the format that is directly pumped out from json-rules-engine.